deploy/initial_schema.sql

```diff
@@ -0,0 +1,78 @@
+BEGIN;
```

Comment:
sqitch generates this? why is the indentation all crazy?

Reply:
sql format, datagrip indented like this by default